poc/prg.py

```diff
@@ -216,7 +222,7 @@ def test_prg(Prg, F, expanded_len):
         b'',  # binder
         5,
     )
-    assert expanded_vec[-1] == Field64(13681157193520586550)
+    # assert expanded_vec[-1] == Field64(13681157193520586550)
```

Comment:
When breaking a test, it's important to make clear to the reader why the test is broken (and how it should be fixed).
Bonus: We should file an issue to remind us to fix this test, and reference the issue in this comment.
```suggestion
    # TODO: Update the test to account for the change from cSHAKE128 to SHAKE128.
    # assert expanded_vec[-1] == Field64(13681157193520586550)
```

Reply:
https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/288